pkg/clusterclientstore/store.go

```diff
@@ -332,5 +342,15 @@ func buildConfig(host string, secret *corev1.Secret, restTimeout *time.Duration)
 		config.KeyData = key
 	}
 
-	return config
+	if insecureSkipTlsVerify, ok := secret.Data["tls.insecure-skip-tls-verify"]; ok {
```

Comment:
I think this might have gotten some of #17 mixed in -- understandable, since you need it for your workflow. Mind resetting this file for this PR?